keyboards/ergodox/keymaps/333fred/Makefile

```diff
@@ -1,7 +1,7 @@
 SUBPROJECT_DEFAULT = infinity
 LCD_BACKLIGHT_ENABLE = yes
 LCD_ENABLE = yes
-LED_ENABLE = yes
+BACKLIGHT_ENABLE = yes
```

Comment:
Any updates on this? I am unable to test it out myself, but I am anxious to get backlighting to work on it again.